plugins/inputs/opcua_listener/subscribe_client.go

```diff
@@ -32,6 +33,54 @@ type SubscribeClient struct {
 	processingCancel context.CancelFunc
 }
 
+func AssignConfigValuesToRequest(req *ua.MonitoredItemCreateRequest, monParams *input.MonitoringParameters, log telegraf.Logger) {
+	req.RequestedParameters.SamplingInterval = float64(time.Duration(monParams.SamplingInterval) / time.Millisecond)
+
+	if monParams.QueueSize != "" {
+		queueSize, err := strconv.ParseUint(monParams.QueueSize, 10, 32)
+		if err == nil {
+			req.RequestedParameters.QueueSize = uint32(queueSize)
+		} else {
+			log.Debugf(
+				"Could not parse queue_size %s into uint32. Standard value %d used instead.",
```

Comment:
I'm inclined to say we should check this earlier and not just log that we could not use their value. Can we do this during init when we load the config?

Reply:
Is this just related to `QueueSize` or the other parameters as well?